db/migrate/20150217121518_create_loc_ids_for_existing_ontologies.rb

```diff
@@ -0,0 +1,35 @@
+class CreateLocIdsForExistingOntologies < ActiveRecord::Migration
+  def up
+    Ontology.all.each do |ontology|
+      ontology.locid = "/#{ontology.repository.path}/"
+      if ontology.parent
+        ontology.locid << "#{ontology.parent.basepath}//#{ontology.name}"
+      else
+        ontology.locid << "#{ontology.basepath}"
+      end
+      ontology.update_column(:locid, ontology.locid)
+
+      ontology.symbols.find_each do |symbol|
+        portion = symbol.name
+        portion = portion[0..-2] if portion.end_with?('>')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.